admin/src/store/store.js

```diff
@@ -0,0 +1,19 @@
+import Vuex from 'vuex'
+import Vue from 'vue'
+
+Vue.use(Vuex)
+
+export const mutations = {
+  token: (state, token) => {
+    state.token = token
+  },
+}
+
+const store = new Vuex.Store({
+  mutations,
+  state: {
+    token: null,
+  },
+})
```

Comment:
If this store reads the same as our frontend, since they share a tld, we need to set a localStorage name for the "cookie". We should expect that both systems can not directly interact with eachother via the users browser.

Please also make a followup ticket for the protection of our localstorage data.

Asume the following:
1. User has his sensitive data in the local storage on `our.tld/wallet`
2 Now a rogue element has access to our server and can place a page at `our.tld/hack`

- Can we make sure that hack cannot access the local storage of wallet?
- What about subdomains?

In the scope of this PR its just required to have two distinctive "cookie" names for admin and wallet.